freeipa/src/main/java/com/sequenceiq/freeipa/configuration/ServiceEndpointConfig.java

```diff
@@ -27,7 +27,7 @@
     @Value("${freeipa.environment.url:}")
```

Comment:
can you please remove the `:` so it always comes from app.yml and in case someone removes it the app will fail